src/components/CircularCountDownTimer.vue

```diff
@@ -343,22 +361,17 @@
         },
         mounted: function () {
             this.$nextTick(() => {
-                this.baseTime = Date.now();
                 this.isMounted = true;
                 if(this.value){
                     const interval = setInterval(function () {
                         if(this.paused){
                             return;
                         }
-                        const delta = Math.floor((Date.now() - this.baseTime) / 1000);
-                        this.value = this.initialValue - delta;
+                        const delta = 1
```

Comment:
There is a possibility that this may won't work properly. For example if user changes the active tab of browser, setInterval won't do anything and this.value will be wrong.

Reply:
It seems to work when the user change tab, I used your component here : http://quizz.be-random.eu/
If you change tab, timer keep going

Reply:
But I've been faced this issue on some browsers.